back/engines/free/email_campaigns/spec/models/email_campaigns/campaigns/manual_project_participants_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EmailCampaigns::Campaigns::ManualProjectParticipants do
+  subject { build(:manual_project_participants_campaign, project: create(:project_with_active_ideation_phase)) }
+
+  describe 'Default factory' do
+    it { is_expected.to be_valid }
+  end
+
+  describe 'validate context_id' do
+    it 'is invalid if context_id is blank' do
+      campaign = build(:manual_project_participants_campaign, context_id: nil)
+      expect(campaign).to be_invalid
+    end
+
+    it 'is invalid if context_id is not a project ID' do
+      campaign = build(:manual_project_participants_campaign, context_id: SecureRandom.uuid)
+      expect(campaign).to be_invalid
+    end
```

Comment:
I'm less sure about this one. It might not work.
```suggestion
    it { is_expected.to validate_inclusion_of(:context_id).in_array(Project.pluck(:id)) }
```

Reply:
Your suggestion doesn't work, but this does:
```Ruby
it { is_expected.to belong_to(:project) }
```
So, I replaced the test in question with that.